etc/openqa/openqa.ini

```
@@ -212,6 +212,14 @@ concurrent = 0
 #list_templates_default_limit = 500
 # Maximum number of templates to include in job_templates api response (to prevent performance issues)
 #list_templates_max_limit = 5000
+# Default number of next jobs to include in ajax request (to prevent performance issues)
+#next_ajax_default_limit = 250
```

Maybe there'd be a better name for this and the other settings. We normally shouldn't care about the "AJAX" part here in this user-facing config file as it is an implementation detail. It would also make sense to have "job" somewhere in these setting keys. So what about:

```suggestion
#next_jobs_default_limit = 250
```

(This suggestion doesn't contain changes for other review comments.)